atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractPersistentStreamStore.java

```diff
@@ -144,10 +144,14 @@ public Sha256Hash apply(StreamMetadata metadata) {
         return hashes;
     }
 
-    protected final StreamMetadata storeBlocksAndGetFinalMetadata(@Nullable Transaction t, long id, InputStream stream) {
+    protected final StreamMetadata storeBlocksAndGetFinalMetadata(@Nullable Transaction t, long id, InputStream stream, boolean computeHash) {
```

Comment:
...well, I see why you're doing that now (using `false` in the generated code). Difficult to immediately see the best solution. Will try and have a look later, or discuss with you when you're online.